### DISCUSSION TITLE
Raspberry pi 5 nextcloudpi container installation failed

### DISCUSSION THREAD

**Discussion by Gismo6303** [tteck/Proxmox#2980] [Help Needed] [answered]
Hello, i've install proxmox on au raspbery pi 5 with the following command :
`sudo apt install proxmox-ve postfix open-iscsi pve-edk2-firmware-aarch64`

Then check if proxmox works, all seams ok, when i try to install the lxc with :
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nextcloudpi-v5.sh)"`

I've the following answer :
 ✗ This script will not work with PiMox!

What does I've done wrong please?

**Comment by tteck** [comment, marked as answer]
These scripts will **NOT** work on `aarch64`.

> `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nextcloudpi-v5.sh)"`

This script does not exist in this repository.
